index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <title>Hien's portfolio</title>
+</head>
+<body>
+    <header class="header">
+        <a href="http://" target="_blank" rel="noopener noreferrer"><img src="/image/logoipsum-265.svg" alt="Logo" class="logo"></a>
+        <nav >
+            <div class="hamberger"></div>
+            <ul class="menu">
+                <li><a href="#portfolio">Porfolio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </nav> 
+    </header>
+    <main>
+        <section class="portfolio" id="portfolio">
+            <h1 class="main-header">I'm Hien <br> Glad to see you!</h1>
+            <p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <div class="social">
+                <p>let's connect</p>
+                <ul class="social-icon">
+                    <li><img src="./image/twister.png" alt="twister" class="contact-icon"></li>
+                    <li><img src="./image/phone.png" alt="phone" class="contact-icon"></li>
+                    <li><img src="./image/mail.png" alt="mail" class="contact-icon"></li>
+                    <li><img src="./image/facebook.png" alt="facebook" class="contact-icon"></li>
+                </ul>
```

Comment:
[Optional] Kindly make the social icons as close to the design as possible using the exact symbols.  @Also, as you may notice, there's no icon for WhatsApp on the figma design but there are Twitter, LinkedIn, Medium, GitHub and Angelist (Now called Wellfound) icons.

Your design:
![image](https://user-images.githubusercontent.com/20554583/204986804-75096301-860e-49cb-9d36-b90a5991e613.png)



Figma design:
![image](https://user-images.githubusercontent.com/20554583/204986649-6ca9d712-d0c2-49e9-9163-4f1acfd94f93.png)